pkg/kubeapiserver/default_storage_factory_builder.go

```diff
@@ -24,9 +24,8 @@ import (
 	"k8s.io/apiserver/pkg/storage/storagebackend"
 )
 
-// SpecialDefaultResourcePrefixes are prefixes compiled into Kubernetes.
-var SpecialDefaultResourcePrefixes = map[schema.GroupResource]string{
-	{Group: "", Resource: "replicationControllers"}:        "controllers",
```

Comment:
ensure https://github.com/kubernetes/kubernetes/blob/master/pkg/registry/core/rest/storage_core.go#L198-L199 doesn't interact with this

Reply:
Yes, does not have any relationship

Reply:
BTW, https://github.com/kubernetes/kubernetes/blob/master/pkg/registry/core/rest/storage_core.go#L198-L199  has all been transformed lowercased by this
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go#L422